README.md

```diff
@@ -59,6 +59,10 @@ The language server for Svelte. Built from [UnwrittenFun/svelte-language-server]
 
 The official vscode extension for Svelte. Built from [UnwrittenFun/svelte-vscode](https://github.com/UnwrittenFun/svelte-vscode) to become the official vscode extension for the language.
 
+#### [`svelte2tsx`](packages/svelte2tsx)
+
+Converts a .svelte file into a legal TypeScript file. Built from [halfnelson/svelte2tsx](https://github.com/UnwrittenFun/halfnelson/svelte2tsx) to provide the auto-complete and import mapping inside the language server.
```

Comment:
url should be https://github.com/halfnelson/svelte2tsx
I wouldn't bother with the attribution, the old repo will be archived.